lang/python-txamqp/Makefile

```diff
@@ -0,0 +1,53 @@
+#
+# Copyright (C) 2009-2015 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=txAMQP
+PKG_VERSION:=0.6.2
+PKG_RELEASE:=1
+PKG_MAINTAINER:=Nate <natester@gmail.com>
```

Comment:
Please consider using your full name in this field to make this more complete and formal.